objax/io/checkpoint.py

```diff
@@ -89,6 +89,8 @@ def restore(self, vc: VarCollection, idx: Optional[int] = None):
             idx: index of the restored checkpoint.
             ckpt: full path to the restored checkpoint.
         """
+        assert isinstance(vc, VarCollection), \
+            f"Must pass a VarCollection to save; received type {type(vc)}."
```

Comment:
```suggestion
        assert isinstance(vc, VarCollection), f'Must pass a VarCollection to restore; received type {type(vc)}.'
```

Nits: (1) use single quotes, (2) we allow 120 character lines in code and (3) I believe this function is for restore, the text was saying save.